core/tests/protractor/accessibility.js

```diff
@@ -41,6 +120,164 @@ describe('screenreader and keyboard user accessibility features', function() {
         .getAttribute('id'));
   });
 
+  it('should move focus to the skip button in the library page',
+    async function() {
+      // Should move the focus to the skip to main content button.
+      await libraryPage.get();
+      await browser.actions().sendKeys('s').perform();
+      var skipButton = element(by.css('.protractor-test-skip-link'));
+      expect(await skipButton.getAttribute('id')).toEqual(
+        await (await browser.driver.switchTo().activeElement())
+          .getAttribute('id'));
```

Comment:
You repeat this pattern a lot. Maybe make this a function to reduce the repetition?